EIPS/eip-4824.md

```diff
@@ -0,0 +1,288 @@
+---
+eip: 4824
+title: Common Interfaces for DAOs
+description: An API standard for decentralized autonomous organizations (DAOs).
+author: Joshua Tan (@thelastjosh), Isaac Patka (@ipatka), Ido Gershtein <ido@daostack.io>, Eyal Eithcowich <eyal@deepdao.io>, Michael Zargham (@mzargham), Sam Furter (@nivida)
```

Comment:
Weird :D done!

Reply:
Hmm, Code Spell shouldn't be failing on people's names.  If it is we should fix that, as it isn't reasonable to add everyone's name to the whitelist I don't think.  Also, I have never seen it fail on a name before.

Reply:
Just dug into this a bit.  I guess the way codespell works is that it looks for words that are "likely misspelled" rather than checking a dictionary.  Normally, names are sufficiently unrelated to words that codespell doesn't detect them as misspellings of other words, but in this case the name is too close to an English word.

I wonder if there is some way we can add a temporary pass without having to permanently add the name to the dictionary?

Reply:
> I wonder if there is some way we can add a temporary pass without having to permanently add the name to the dictionary?

... remove codespell.

Reply:
It is quite useful, it catches a *lot* of typos.

Reply:
I can also ask Sam @nivida to change his name...

Reply:
@MicahZoltu @Pandapip1 Hi folks, I think this is the only thing holding back publishing; we're already starting to roll out additional support + integrating with Moloch v2 & v3. Would love your help getting this through 🙏